packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -920,6 +921,18 @@ addFieldsDict(Posts, {
     }
   },
 
+  sharingSettings: {
```

Comment:
This needs a `hidden: true`; as it is now, it adds a visible "Sharing Settings" field to the new-post form, above the title.